android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1170,6 +1178,9 @@ protected void deployNextRouterService(){
 	}
 	
 	public void startUpSequence(){
+
+		setRouterServiceExceptionHandler();
```

Comment:
I think we should call this method first thing in `onCreate()` because currently, the service could enter foreground before adding the exception handler